### PR TITLE
Add privacy policy link to footer

### LINKED ISSUE #445: Add link to Privacy Policy
## Describe the solution you'd like
Add a link in the footer to the [RAC Privacy Policy](https://rockarch.org/about-us/privacy-policy/).

### PR DESCRIPTION
**Pull request by bonniegee**
Fixes #445.

### CODE REVIEW

**Comment by HaSistrunk** [comment]
I think merge to `development` first?

File: aurora/bag_transfer/templates/transfers/footer.html [hunk 1]
```diff
@@ -4,5 +4,6 @@
   	<img src="{% static "dist/img/RAC-logo.png" %}" alt="The Rockefeller Archive Center">
   </a>
   &copy; 2017 Rockefeller Archive Center. Aurora code available under the
-  <a href="https://github.com/RockefellerArchiveCenter/aurora/blob/master/LICENSE">MIT License.</a>
+  <a href="https://github.com/RockefellerArchiveCenter/aurora/blob/master/LICENSE">MIT License.</a>
```
Comment (HaSistrunk) [inline aurora/bag_transfer/templates/transfers/footer.html]:
```suggestion
  <a href="https://github.com/RockefellerArchiveCenter/aurora/blob/base/LICENSE">MIT License.</a> 
```

Comment (HaSistrunk) [reply aurora/bag_transfer/templates/transfers/footer.html]: I just see that we never updated this license link when we changed from master to base, so looks like a good time to update!

**Comment by bonniegee** [comment]
> I think merge to `development` first?

It looks like `development` had its last commit a couple of years ago. @helrond , can the `development` branch be deleted? Should this point at a release?

**Comment by helrond** [comment]
Let's point this at the `v2.1` branch. I'm going to delete `development` - you're right, it's way out of date.

**Comment by bonniegee** [comment]
@helrond I changed the base; because I branched off of base it's pulling in the commits related to instructions for Windows users. Is this okay or should I delete this branch and branch off of `v2.1`?

**Comment by helrond** [comment]
grarararar. I think let's deal with the merge conflicts later, so let's base this off of 2.1

**Comment by bonniegee** [comment]
Okay--things went haywire when I tried to rebase locally, so I'm going to delete this PR/branch and push everything back up.

**Pull request closed without merging**